static/app/utils/replays/hooks/useExtractedCrumbHtml.tsx

```diff
@@ -28,78 +28,102 @@ type HookOpts = {
   replay: ReplayReader;
 };
 
+const requestIdleCallback =
+  window.requestIdleCallback ||
+  function (cb) {
```

Comment:
We should give this function a name

```suggestion
  function requestIdleCallbackPolyfill(cb) {
```